docs/glossary/index.md

```diff
@@ -0,0 +1,355 @@
+
+### incentivized block
+
+A block that is eligible for a reward for the validator that signs it.
+
+## J
+
+
+
+## K
+
+
+
+## L
+
+### launch information
+
+The set of information relied on to deterministically launch the network of a chain, including the information required to generate the genesis file and the list of persistent peers for the network.
+
+### last block height
+
+The height of the last block that was monitored in a monitoring round.
+
+### last reward height
+
+The block height of the last incentivized block in a testnet for a reward pool.
+
+### light node 
+
+A program that processes only block headers and a small subset of transactions. 
+
+## M
+
+### mainnet
+
+Short for main network, the original and functional blockchain where actual transactions take place in the distributed ledger.
+
+### module
+
+A subset of the state-machine. A Cosmos SDK app is usually composed of an aggregation of modules.
+
+### monitoring packet 
+
+An atomic IBC packet that is sent from a testnet to SPN that contains data from the monitoring period.
+
+### monitoring period
+
+A number of blocks specific to the testnet to monitor validator activities. 
+
+### monitoring round
+
+The monitored validator activities during a round. A round is a variable number of blocks. A round ends when the validator set of the testnet changes or when the monitoring period ends.
+
+### minting 
+
+The process of creating new units of cryptocurrency tokens.
+
+### mnemonic
+
+A sequence of words that is used as seed to derive private keys. The mnemonic is at the core of each wallet.
+
+## N
+
+### node
+
+Any computer that is connected to a blockchain network is referred to as a node.
+
+## O
+
+### oracle
+
+A communication module that has built-in compliance with IBC protocol to query data points of various types.
+
+## P
+
+### pool reward
+
+The percentage of reward for a given reward pool that is distributed during a distribution round. The percentage depends on the current reward height and the last reward height of the reward pool.
+
+## Q
+
+
+
+## R
+
+### refund per pool
+
+The percentage of reward that is refunded to the reward provider during a distribution round. The percentage for each reward pool depends on the number of unsigned blocks.
+
+### relaying tip
+
+The reward offered for relaying a monitoring packet.
+
+### relaying tip pool
+
+A pool that contains relaying tips. The pool determines how the tips are distributed by defining the number of tips per block. Distribution instructions cannot be modified, but the tips contained in the pool can be refunded to the provider. A testnet can have any number of associated pools. 
+
+### request 
+
+Solicit a change in the launch information of the chain. To be effective, a request must be approved by a coordinator. 
```

Comment:
Very contextualized definition. While reading `request` I actually expected a definition of the cosmos chain  requests.

Reply:
Like it's done for validator, the definition should be suffixed with `, Ignite chain`

Reply:
Same for other definitions below